src/ipc/ipc-helper.c

```diff
@@ -40,8 +40,8 @@ struct comp_buffer *buffer_new(const struct sof_ipc_buffer *desc)
 {
 	struct comp_buffer *buffer;
 
-	tr_info(&buffer_tr, "buffer new size 0x%x id %d.%d flags 0x%x",
-		desc->size, desc->comp.pipeline_id, desc->comp.id, desc->flags);
```

Comment:
Keep this as well.

Reply:
I reverted to info, this is pipeline allocate buffer, always see it, personal think, it is not so important.